crates/language/src/language.rs

```diff
@@ -536,6 +536,14 @@ mod tests {
         assert_eq!(subbed, "µfoo$('µbar')");
     }
 
+    #[test]
+    fn test_php_substitute_variable() {
+        let snippet = "$foo$('$bar')";
+        let lang = Php::new(None);
+        let subbed = lang.substitute_metavariable_prefix(snippet);
+        assert_eq!(subbed, "µ[foo]$('µ[bar]')");
+    }
```

Comment:
Move this test to the php file, also this test isn't quite right. you correctly added:
`grit_metavariable: ($) => token(prec(PREC.GRIT_METAVARIABLE, choice('µ...', /µ[a-zA-Z_][a-zA-Z0-9_]*/)))`
to the metavariable grammar, but notice that the node wouldn't match here, since the brackets aren't valid. The issue with php is not on the replaced side, but on the user side. since `$identifier` is already a variable in php, so it can't be a metavariable in grit. instead we should use a different prefix that's not already used. Maybe `#` would be a good one, I don't think it's used much in php. So the test instead should show that:

`"#foo$('$bar')"` is replaced with: `"µfoo$('$bar')"`
(bar is a php variable, not a grit metavariable)